builtin/game/chat.lua

```diff
@@ -621,6 +621,10 @@ core.register_chatcommand("set", {
 
 		setname, setvalue = string.match(param, "([^ ]+) (.+)")
 		if setname and setvalue then
+			if string.match(setname, "^secure.") then
```

Comment:
```suggestion
			if setname:sub(1, 7) == "secure." then
```

Reply:
Also fixes bug where I forgot to escape the `.` ;)